CHANGELOG.md

```diff
@@ -6,40 +6,39 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [3.0.0-beta.0] - 2022-02-08
-    
+
 ### Added
-    
+- Add static property `audioBufferSourceNode` in `DefaultDeviceController` to store the reference to the Web Audio node which is used to create an empty audio device. And in the `DefaultDeviceController.destroy()` method clean up this Web Audio resource to prevent memory leaks ([#issue 1660](https://github.com/aws/amazon-chime-sdk-js/issues/1660)).
+
 ### Removed
 - Remove support for Plan B as well as Safari (and iOS) 12+. The minimum Safari and iOS supported version is now 13. Also clean up all plan-B code path.
 - Remove all deprecated meeting status code.  
-    
+
 ### Changed
 - Add support for pre-release in Versioning.
 - Removed upward BWE throttling logic in VideoPriorityBasedPolicyConfig, which was increasing recovery time more then intended, whereas its main focus was towards slowing downturns in BWE when the network is actually stable. We may come back to configuring the recovery delay another time.
 - Add support for aws-sdk js v3 for messaging session.
-    
+
 ### Fixed
 - Fix a worker resource leak with `BackgroundBlurProcessor` and `BackgroundReplacementProcessor`.
-    
+
 ## [2.27.0] - 2022-01-27
-    
+
```

Comment:
Why is there a line here? It looks like below we have no space between Added and version.

Reply:
It is best to have the `Heading`s surrounded by blank lines according to markdown lint. But this is not mandatory.
This change is to remove the unnecessary spaces/tabs.

Reply:
Gotcha, that makes sense. I just noticed it wasn't consistent but looks like you updated the other ones now.

Reply:
Yes, I usually clean up when I raise a PR. But there are still a lot of small issues, and they are increasing all the time. I can not always update them, but it's better than not doing anything.

Reply:
> It is best to have the Headings surrounded by blank lines according to markdown lint. But this is not mandatory.
This change is to remove the unnecessary spaces/tabs.

According to? Anyway, its better to fix it in the script that generates it https://github.com/aws/amazon-chime-sdk-js/blob/main/script/version-utils.js#L64.

Reply:
I saw this rule in markdown lint and think it's a good practice to follow and can make the style of our CHANGELOG raw code consistent, so I usually do a clean up once a while when I raise a PR. ([markdown lint](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md022))

It's just a VSCode plugin, so it's not that official. But I observe we did this at the beginning when creating the CHANGLOG, so just want to follow it.

I will push a new commit to update that script.